Components/Sources/Components/Style/WKIcon.swift

```diff
@@ -66,85 +67,93 @@ public enum WKSFSymbolIcon {
         case .exclamationMarkCircle:
-            return UIImage(systemName: "exclamationmark.circle", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "exclamationmark.circle", withConfiguration: configuration)
         case .textFormatSuperscript:
-            return UIImage(systemName: "textformat.superscript", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "textformat.superscript", withConfiguration: configuration)
         case .textFormatSubscript:
-            return UIImage(systemName: "textformat.subscript", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "textformat.subscript", withConfiguration: configuration)
         case .underline:
-            return UIImage(systemName: "underline", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "underline", withConfiguration: configuration)
         case .strikethrough:
-            return UIImage(systemName: "strikethrough", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "strikethrough", withConfiguration: configuration)
         case .multiplyCircleFill:
-            return UIImage(systemName: "multiply.circle.fill", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "multiply.circle.fill", withConfiguration: configuration)
         case .chevronRightCircle:
-            return UIImage(systemName: "chevron.right.circle.fill", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate).imageFlippedForRightToLeftLayoutDirection()
+            image = UIImage(systemName: "chevron.right.circle.fill", withConfiguration: configuration)?.imageFlippedForRightToLeftLayoutDirection()
         case .close:
-            return UIImage(systemName: "multiply", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "multiply", withConfiguration: configuration)
         case .ellipsis:
-            return UIImage(systemName: "ellipsis", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "ellipsis", withConfiguration: configuration)
         case .pencil:
-            return UIImage(systemName: "pencil", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
+            image = UIImage(systemName: "pencil", withConfiguration: configuration)
         case .plusCircleFill:
-            return UIImage(systemName: "plus.circle.fill", withConfiguration: configuration)?.withRenderingMode(.alwaysTemplate)
-
+            image = UIImage(systemName: "plus.circle.fill", withConfiguration: configuration)
+        }
+        
+        image = image?.withRenderingMode(.alwaysTemplate)
```

Comment:
Was the idea to use the `renderingMode` method argument here?

Reply:
@mazevedofs Good catch! Originally I thought I might need to be able to pass different rendering modes to support both palette and monochrome icons, but it turns out `.alwaysTemplate` worked for both. I forgot to remove the parameter after that. I'll add that cleanup to my next PR.